core/src/test/resources/rewrite/RedundantBraces.stat

```diff
@@ -192,6 +192,14 @@ object a {
 object a {
   s"Hello ${foo}_bar"
 }
+<<< not redundant enclosing braces in str interpolation (right part starts with special symbol)
```

Comment:
Can we move all of the "in str interpolation" tests into a separate file named `RedundantBracesStrInterpolation.stat`? Also maybe remove the "redundant braces" part since it's implicit for that test suite, I prefer to keep test names as short as possible.